code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -296,3 +296,10 @@
 /obj/item/weapon/storage/box/syndie_kit/mimery/PopulateContents()
 	new /obj/item/weapon/spellbook/oneuse/mimery_blockade(src)
 	new /obj/item/weapon/spellbook/oneuse/mimery_guns(src)
+
+/obj/item/weapon/storage/box/syndie_kit/holoparasite
+	name = "box"
+
+/obj/item/weapon/storage/box/syndie_kit/holoparasite/PopulateContents()
+	new /obj/item/weapon/guardiancreator/tech/choose/traitor(src)
```

Comment:
I suggest double-checking that this cannot be used by changelings and that it can be used to select specific guardian types. I don't quite remember what the `traitor` guardiancreator does.